### PR TITLE
chore: import x-translations

### PR DESCRIPTION
**Pull request by pmareke**
EX-4362

### CODE REVIEW

File: packages/x-translations/.versionrc.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
Comment (guillei10) [inline packages/x-translations/.versionrc.js:1]: Do we actually need this file? Been checking the other packages an none of them have a `.versionrc.js` file.

- [ ] Check if we need this file.

**Review by guillei10 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: cypress/browsers:node12.18.3-chrome87-ff82
+    container: cypress/browsers:node14.17.0-chrome91-ff89
```
Comment (javieri-empathy) [inline .github/workflows/build.yml:6]: Change the container in the other 3 workflows too

File: packages/x-translations/package.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@empathyco/x-translations",
+  "version": "1.0.0",
+  "description": "A project which allow the user to import and export the translations.",
```
Comment (javieri-empathy) [inline packages/x-translations/package.json:4]: Fix the `allow` -> `allows` typo

File: packages/x-translations/package.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/empathyco/x.git",
+    "directory": "packages/x-translations"
+  },
+  "scripts": {
+    "build": "tsc && npm pack",
+    "lint": "eslint src",
+    "release": "standard-version --no-verify",
```
Comment (javieri-empathy) [inline packages/x-translations/package.json:23]: Remove the release scripts

File: packages/x-translations/package.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    "package.json",
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/empathyco/x.git",
+    "directory": "packages/x-translations"
+  },
+  "scripts": {
+    "build": "tsc && npm pack",
```
Comment (javieri-empathy) [inline packages/x-translations/package.json:21]: Add the `npm pack` script in the `postbuild` script.

File: packages/x-translations/package.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  "dependencies": {
+    "@empathyco/x-deep-merge": "^1.2.2-alpha.5"
+  },
+  "devDependencies": {
+    "@empathyco/eslint-plugin-x": "^1.2.0",
+    "@types/jest": "~25.2.1",
+    "@types/node": "~14.14.39",
+    "commitizen": "~4.0.3",
+    "cz-conventional-changelog": "~3.1.0",
+    "husky": "~4.3.5",
```
Comment (javieri-empathy) [inline packages/x-translations/package.json:36]: Remove `husky`, and the `lint-staged` packages, and its sections below

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: packages/x-translations/.versionrc.js (hunk 1 above)
Comment (guillei10) [inline packages/x-translations/.versionrc.js:1]: Do we need this file? Been checking other packages and cannot find this file in them.

**Review by guillei10 — CHANGES_REQUESTED** [review]

**Review by guillei10 — DISMISSED** [review]

**Review by javieri-empathy — APPROVED** [review]

**Review by guillei10 — APPROVED** [review]

**Review by javieri-empathy — APPROVED** [review]

**Pull request merged**